test_scripts/API/SystemRequest_backslash/ATF_N_SystemRequest_FileName_Contains_BackSlashScreened_as_Param.lua

```diff
@@ -0,0 +1,69 @@
+-- Requirement summary:
+--[GENIVI] SDL must respond INVALID_DATA in case mobile app sends "/" (backslash) symbol at "fileName" parameter
```

Comment:
@Aleksandriys, summary and description is about backslash, but slash "/" symbol is used; so requirement summary and description should be updated with correct backslash symbol.